bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/AbstractRemoteAddonService.java

```diff
@@ -298,11 +298,9 @@ protected boolean remoteEnabled() {
         try {
             Configuration configuration = configurationAdmin.getConfiguration("org.openhab.addons", null);
             Dictionary<String, Object> properties = configuration.getProperties();
-            if (properties == null) {
-                // if we can't determine a set property, we use true (default is remote enabled)
-                return true;
-            }
-            return ConfigParser.valueAsOrElse(properties.get(CONFIG_REMOTE_ENABLED), Boolean.class, true);
+            // if we can't determine a set property, we use true (default is remote enabled)
```

Comment:
TBH, I don't think this is an improvement. Looking at the code before I can easily determine what it does, now I have to think about it.